cmd/nvidia-ctk/runtime/configure/configure.go

```diff
@@ -234,11 +235,13 @@ func (m command) configureConfigFile(c *cli.Context, config *config) error {
 		cfg, err = containerd.New(
 			containerd.WithLogger(m.logger),
 			containerd.WithPath(configFilePath),
+			containerd.WithReference(pkgconfig.FromCLI([]string{"containerd", "config", "dump"})),
```

Comment:
nit: Ideally this should be configurable.

Reply:
So you mean adding an options struct specific to  the `FromCLI` implementation?

Reply:
No, I mean that we should be able to select pulling the config from a file or from the CLI (where we decide what the command is). Something like:
```
--config-source [ file | command | empty]
```

Reply:
This is done